build

```diff
@@ -39,14 +38,13 @@ try {
 	if (err.code !== 'MODULE_NOT_FOUND') throw err; // should never happen
 
 	console.log('config.js does not exist. Creating one with default settings...');
+	fs.mkdirSync('config', {recursive: true});
```

Comment:
...AI code?

My main concern is that it'll clutter up the diff with useless changes like this one.

```suggestion
```